features/tab-access-tabs.feature

```diff
@@ -0,0 +1,79 @@
+Feature: Get or set TabStops
+  In order to customize tab stops
+  As a python-docx developer
+  I need a TabStops object with read/write tab_stop properties
+
+
+  @wip
+  Scenario Outline: Get tab_stop objects
+    Given a paragraph having <value> tab_stop elements
+     Then paragraph_format.tab_stops contain tab_stop objects
+      
+    Examples:  values for tab_stop elements 
+      | value |
+      | some  |
+      
+
+  @wip
+  Scenario Outline: Count tab stop objects
+    Given a paragraph having <value> tab_stop elements
+      Then paragraph_format.tab_stops has <lenval> length
+      
+    Examples:  length values for tab_stop elements
+      | value | lenval |
+      | no    | 0      |
+      | some  | 3      |
+      
+
+  @wip
+  Scenario Outline: Iterate over tab stop objects
+    Given a paragraph having <value> tab_stop elements
+     Then I can iterate over the tab_stop elements
+      
+    Examples:  values for tab_stop elements 
+      | value |
+      | some  |
+
+
+  @wip
+  Scenario Outline: Access a tab stop by index
+    Given a paragraph having <value> tab_stop elements
+     Then I can access a tab_stop by index
+      
+    Examples:  values for tab_stop elements 
+      | value |
+      | some  |
+
+
+  @wip
+  Scenario Outline: Add a tab stop
+    Given a paragraph having <value> tab_stop elements
+     When I add a tab_stop
+     Then paragraph_format.tab_stops has <lenval> length
+      
+    Examples:  values for tab_stop elements 
+      | value | lenval |
+      | no    | 1      |
+      | some  | 4      |
+
+
+  @wip
+  Scenario Outline: Delete a tab stop
+    Given a paragraph having <value> tab_stop elements
+     When I delete a tab_stop
+     Then paragraph_format.tab_stops has <lenval> length
+      
+    Examples:  values for tab_stop elements 
+      | value | lenval |
+      | some  | 2      |
+
+
+  @wip
+  Scenario Outline: Clear tab stops
+    Given a paragraph having <value> tab_stop elements
+     When I clear the tab_stops
+     Then paragraph_format.tab_stops has <lenval> length
+     
+    Examples:  values for tab_stop elements 
+      | value | lenval |
+      | some  | 0      |
```

Comment:
This can become a Scenario too since there is only one set of values. So would become something like:

```cucumber
Given a tab_stops object containing 3 tab stops
 When I call tab_stops.clear()
 Then len(tab_stops) is 0
```